cli/ballerina-cli/src/main/java/io/ballerina/cli/launcher/util/BalToolsUtil.java

```diff
@@ -231,4 +237,59 @@ private static List<File> findJarFiles(File directory) {
         }
         return jarFiles;
     }
+
+    /**
+     * Update the bal-tools.toml file with the version and the active flags.
+     * bal-tools.tomls of updates 6, 7  only has id, name and org fields. Therefore, we need to update the
+     * bal-tools.toml file when the user moves from updates 6, 7 to update 8 and above.
+     */
+    public static void updateOldBalToolsToml() {
```

Comment:
If a user switches to U8 and switch back to an older update, will the older update break?

Reply:
No. But identified an issue where a user switches from U7->U8->U7->U8 will cause to not have any active version for some tools. Will fix that

Reply:
Now with the new commit the changes U7<->U8 are working fine